### DISCUSSION TITLE
Question: Is there a way to customise the delete animation for ListView?

### DISCUSSION THREAD

**Discussion by harvinders** [microsoft/microsoft-ui-xaml#7446] [Q&A]
ListView provide nice [AddDeleteThemeTransitions](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Media.Animation.AddDeleteThemeTransition?view=winrt-20348) , however, I would like to customise the delete animation to be something like swipe out from the right edge. The way notifications are dismissed from the notification view of windows 10. 

Is it possible?

**Comment by StephenLPeters** [comment]
@stmoy are these transitions customizable?

@harvinders ListViewItems have a hand crafted template from code behind using internal api's for performance reasons. You can provide your own template to these classes, however it wont come with these performance gains. I suspect that this is difficult but possible to do however it would involve providing a new template and losing these performance gains, which depending on your situation could be varying amounts of bad.
